suite/fuzz/fuzz_disasm.c

```diff
@@ -189,14 +189,24 @@ static struct platform platforms[] = {
         CS_MODE_BIG_ENDIAN,
         "tms320c64x"
     },
-#if CS_NEXT_VERSION >= 5
```

Comment:
I think the ifdef needs to remain so that both `next` and `master` branch can be fuzzed

Reply:
Oh I found both master and next branch has the same `CS_NEXT_VERSION = 5` that's why I removed it.

Reply:
Ok thanks @david942j so your code is good

@aquynh how can a program know at compile time which architectures are supported by the current capstone ie branch `master` and branch `next` ?